packages/jest-cli/src/reporters/summary_reporter.js

```diff
@@ -154,7 +152,18 @@ export default class SummaryReporter extends BaseReporter {
         globalConfig,
         updateCommand,
       );
-      snapshotSummary.forEach(this.log);
+
+      const failedTests = aggregatedResults.numFailedTests;
+      const runtimeErrors = aggregatedResults.numRuntimeErrorTestSuites;
+
+      //don't mention any obsolete snapshots when test fails
+      snapshotSummary.forEach(status => {
+        //if status contains the word obsolete and the test have failed do
+        //nothing otherwise log status.
+        if (status.search('obsolete') != -1 && failedTests + runtimeErrors > 0)
```

Comment:
Can you extract `failedTests + runtimeErrors > 0` into a variable above the loop?